### DISCUSSION TITLE
Can't get Traefik2 dashboard working on k3s

### DISCUSSION THREAD

**Discussion by crawler-n** [techno-tim/techno-tim.github.io#91] [Questions]
Issue: Can't get Traefik2 dashboard working on k3s and rancher

Background:

- I am following Tim's guide listed here -> https://techno-tim.github.io/posts/k3s-traefik-rancher/
- When choosing kubernetes ingress or ingressroute option, can't get the dashboard working

Potential Solution:

- In the `https://github.com/techno-tim/techno-tim.github.io/blob/master/reference_files/traefik2-k3s-rancher/config/traefik-dashboard-ingressroute.yml` file, annotation needs to be added for ingressClass.

Please comment or let me know if this is not necessary or if I am making a mistake somewhere.

**Comment by timothystewart6** [comment]
If you're not getting any responses here in the discussions tab, feel free to join our Discord for questions! https://discord.gg/DJKexrJ
